src/websocket/run.py

```diff
@@ -1,13 +1,18 @@
-"""Entry point for the Websocket API."""
+"""Entry point for the WebSocket API."""
 
-import asyncio
 import logging
-from src.websocket.websockets_server import WebSocketServer
+
+import uvicorn
+
+logger = logging.getLogger(__name__)
 
 
 def run_websocket_api(websocket_port, host):
-    """Starts the websocket server."""
-    logger = logging.getLogger(__name__)
+    """Starts the WebSocket API server."""
     logger.info(f"Starting Websockets server on '{host}:{websocket_port}'")
-    web_socket_server = WebSocketServer(port=websocket_port, host=host)
-    asyncio.run(web_socket_server.start_server())
+    uvicorn.run(
+        "src.websocket.websockets_server:app",
```

Comment:
Does this have to be a string like this?  This sounds like it would break easily.

If there is no other way to do this just close this comment

Reply:
Yes unfortunately